innovation_sweet_spots/analysis/notebooks/foodtech/research_funding/research_step3_analysis.py

```diff
@@ -547,17 +565,16 @@
         ),
         width=400,
     )
-    .mark_line(
-        interpolate="monotone",
-        size=3,
-    )
+    .mark_line(interpolate="monotone", size=3)
     .encode(
         x=alt.X("year:O"),
         y=alt.Y("no_of_projects:Q", title="Number of new projects"),
-        color=alt.Color("Category:N", scale=alt.Scale(domain=domain, range=range_)),
+        color=alt.Color(
+            "Category:N",
+            scale=alt.Scale(domain=domain_excl_health, range=range_excl_health),
+        ),
```

Comment:
This and similar plots were plotting with a legend containing `Health` but without any data for `Health`. I have added variables `domain_excl_health` and `range_excl_health` so the charts' legends match the data.